### PR TITLE
Partial TL translations for 3-0-migration-guide.rst - 1,148 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
This is a continuation of my work for this file. Please help me review.

### CODE REVIEW

File: tl/appendices/3-0-migration-guide.rst [hunk 1]
```diff
@@ -476,178 +476,198 @@ Tugon
...
+==========
+
+Ang sesyon na class ay hindi na static, sa halip ang sesyon ay maaaring i-access
+gamit ang kahilingan na object. Tingnan ang :doc:`/development/sessions` na
+dokumentasyon para sa paggamit ng sesyon na object.
+
+* Ang :php:class:`Cake\\Network\\Session` at may kaugnayang sesyon na mga class
+  ay nailipat sa ilalim ng ``Cake\Network`` na namespace.
+* Ang ``SessionHandlerInterface`` ay itinanggal sa pabor ng isang ibinigay ng
+  PHP.
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: PHP to PHP mismo

File: tl/appendices/3-0-migration-guide.rst [hunk 2]
```diff
@@ -476,178 +476,198 @@ Tugon
...
+* Ang katangian na ``Session::$requestCountdown`` ay itinanggal.
+* Ang sesyong checkAgent na tampok ay itinanggal. Ito ay nagsanhi ng ilang mga
+  bug kapag nag-frame ang chrome, at hindi kasali ang flash player.
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: hindi kasali ang flash player to sangkot ang flash player

File: tl/appendices/3-0-migration-guide.rst [hunk 3]
```diff
@@ -476,178 +476,198 @@ Tugon
...
 =============
 
-* ``HttpSocket`` is now :php:class:`Cake\\Network\\Http\\Client`.
-* Http\Client has been re-written from the ground up. It has a simpler/easier to
-  use API, support for new authentication systems like OAuth, and file uploads.
-  It uses PHP's stream APIs so there is no requirement for cURL. See the
-  :doc:`/core-libraries/httpclient` documentation for more information.
+* Ang ``HttpSocket`` ngayon ay :php:class:`Cake\\Network\\Http\\Client` na.
+* Ang Http\Client ay muling naisulat mula sa panimula paitaas. Ito ay mayroong
+  isang mas simpleng/mas madaling magamit na API, suporta para sa bagong
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: mas simpleng to mas simple

File: tl/appendices/3-0-migration-guide.rst [hunk 4]
```diff
@@ -476,178 +476,198 @@ Tugon
...
+  subclass, ang kumpigurasyon lamang sa subclass ang magagamit.
+- Ang ``Controller::httpCodes()`` ay tinanggal, sa halip ay gamitin ang
+  :php:meth:`Cake\\Network\\Response::httpCodes()`.
+- Ang ``Controller::disableCache()`` ay tinanggal, sa halip ay gamitin ang
+  :php:meth:`Cake\\Network\\Response::disableCache()`.
+- Ang ``Controller::flash()`` ay tinanggal. Ang paraang ito ay bihira lamang
+  ginamit sa tunay na mga aplikasyon at walang nang layunin na pinagsisilbihan.
+- Ang ``Controller::validate()`` at ``Controller::validationErrors()`` ay
+  tinanggal. Sila ay mga tirang mga paraan mula sa 1.x na kapanahunan kung saan
+  ang mga alalahanin ng mga model + mga controller ay malayong mas magkaakibat.
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: model to modelo

File: tl/appendices/3-0-migration-guide.rst [hunk 5]
```diff
@@ -476,178 +476,198 @@ Tugon
...
+  paraan ay isang aksyon o hindi. Ang pagbabagong ito ay ginawa upang payagan
+  ang mas madaling pag-customize ng kung ano at kung ano ang hindi ang binibilang
+  bilang isang aksyon.
+- Ang ``Controller::$Components`` na katangian ay tinanggal at pinalitan ng 
+  ``_components``. Kung kailangan mong mag-load ng mga komponent sa runtime dapat
+  kang gumamit ng ``$this->loadComponent()`` sa iyong controller.
+- Ang lagda ng :php:meth:`Cake\\Controller\\Controller::redirect()` ay binago
+  sa ``Controller::redirect(string|array $url, int $status = null)``. Ang 
+  pangatlong argumento na ``$exit`` ay tinanggal. Ang paraan ay hindi na nagpapadala
+  ng tugon at exit na iskrip, sa halip ito ay nagsasauli ng isang ``Response``
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: exit to palabas

File: tl/appendices/3-0-migration-guide.rst [hunk 6]
```diff
@@ -476,178 +476,198 @@ Tugon
...
-  successful identification and then use ``setUser()`` to save the info to
-  session for persistence across requests.
-
+- Ang ``Default`` ngayon ay ang default na password hasher na ginagamit ng pagpapatunay
+  na mga class. Ito ay eksklusibong gumagamit ng bcrypt hashing na algoritmo. Kung 
+  gusto mong magpatuloy sa paggamit ng SHA1 hashing na ginamit sa 2.x gamitin ang 
+  ``'passwordHasher' => 'Weak'`` sa iyong authenticator na kumpigurasyon.
+- Isang bagong ``FallbackPasswordHasher`` ang dinagdag upang tulungan ang mga gumagamit
+  na maglipat ng lumang mga password mula sa isang algoritmo patungo sa iba pa. Suriin
+  ang dokumentasyon ng AuthComponent para sa karagdagang info.
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: info to impormasyon

**Comment by mind-blast** [comment]
Please try to check my suggested words. Thank you.

File: tl/appendices/3-0-migration-guide.rst (hunk 5 above)
Comment (josteinfrancois) [reply tl/appendices/3-0-migration-guide.rst]: I used "labasan" for this instead. the word "palabas" could give another meaning.

**Comment by josteinfrancois** [comment]
Hey @robin-ho . I have edited the pull request based on your suggestions. Thank you.

**Pull request merged**